[Feature] Add navigation methods to WebView for use in FuseJS

DerMitch opened the pull request:
This patch exports four methods of WebViews to allow them to be controlled by FuseJS code. This allows the creation of UI elements or logic to build a mini-webbrowser, which can be sometimes useful in apps.

This PR contains:
- [X] Changelog
- [ ] Documentation
Are the code comments enough?
- [ ] Tests
If needed, some guidance would be appreciated as I'm pretty new to Fuse/Uno development.

kusma commented on CHANGELOG.md:
```diff
@@ -80,6 +80,8 @@
 Fixed issue where custom URI schemes were matched too greedily in URLs, making for erroneously intercepted URL requests.
 
+- Exported the methods goBack, goForward, reload and stop for use in FuseJS
+
```
This is in the wrong section of the changelog. This feature is not part of 1.3, which has been feature-frozen for a while.

kusma replied:
Just to be clear, this should be put in the "Unreleased"-section of the changelog. I like that you looked for a suitable heading, but sadly there isn't one that we can reuse for this :/

kusma commented on Source/Fuse.Controls.WebView/WebView.ScriptClass.uno:
```diff
@@ -47,6 +51,47 @@ namespace Fuse.Controls
+		static void stop(Context c, WebView view, object[] args)
+		{
+			view.Stop();
+		}
```
It seems to me like the other script-methods verify their arguments and raise a UserError if they're not the expected lenght. Any reason why these methods don't?

DerMitch commented:
Hey @kusma, thank you for the quick response.

I moved the section in the CHANGELOG.md and added argument validation. As the methods don't take any arguments, I didn't saw a reason at first to add further checks.

As 1.3 is in feature freeze, I assume this might be released in 1.4 then?

kusma commented on CHANGELOG.md:
```diff
@@ -5,6 +5,10 @@
+### WebView
+- Exported the methods goBack, goForward, reload and stop for use in FuseJS
+
+# 1.2
```
I doubt you meant to add this heading...

DerMitch commented:
Oh sorry, I think something went wrong when rebasing from upstream master. 😞 

Patch should be clean now.
